### ISSUE TITLE
Release a new package with webpack 5 fix to npm packages

### ISSUE THREAD

**Issue by Oggy107** [Semantic-Org/Semantic-UI-CSS#81]
Release a new package with webpack 5 fix to npm packages

**Comment by imraazrally** [comment]
It looks like the double semi-colon issue was fixed in the codebase but a new package hasn't been released yet. I pointed my dependency file (package.json) directly to build from the git-repo like this 
```
"semantic-ui-css": "git+https://github.com/Semantic-Org/Semantic-UI-CSS.git"
```

**Comment by mraible** [comment]
I can confirm that this fix solves https://github.com/Semantic-Org/Semantic-UI-CSS/issues/82 for me.

```json
"semantic-ui-css": "git+https://github.com/Semantic-Org/Semantic-UI-CSS.git"
```

**Comment by firrae** [comment]
Would be nice if this was pushed out. It's making me serious rethink using Semantic UI if fixes like this don't get version pushes.

**Comment by Oggy107** [comment]
> Would be nice if this was pushed out. It's making me serious rethink using Semantic UI if fixes like this don't get version pushes.

exactly... I can't get the reason why they are not pushing it to npm.

**Comment by Lord-Mendoza** [comment]
For my project I ended up cloning this repo then republishing it on npm. You guys can just do the same for now. I agree they should just release a new version considering how simple change it needs.

**Comment by diego-iturriaga** [comment]
We are having the same issue in our project, when is it planned to release to npm?

**Comment by 736986614** [comment]
شكرا

**Comment by jlukic** [comment]
Id be willing to fix this if someone can provide a clear explanation of the issue. There's a fix in main that is not in the npm package?
